pkg/importer/dataStream.go

```diff
@@ -918,51 +849,39 @@ func (d *DataStream) convertQcow2ToRaw(src, dest string) error {
 }
 
 // Copy endpoint to dest based on passed-in reader.
-func (d *DataStream) copy(dest string) (err error) {
+// if the error is not nil and true is return, we require scratch space to complete the copy.
+func (d *DataStream) copy(dest string) (bool, error) {
+	if d.HasScratchAvailable {
+		defer CleanDir(d.ScratchDataDir)
```

Comment:
If the scratch PVC has a distinct lifecycle from the pod, may make sense to preemptively clean up the scratch dir as well just in case.

Reply:
I am not sure I follow, we linked the pod and scratch pvc lifecycle at this point. This code is to clean up the scratch space after we are done with it.